nixos/modules/services/misc/octoprint.nix

```diff
@@ -128,6 +134,8 @@ in
       };
     };
 
+    networking.firewall = mkIf cfg.openFirewall {
```

Comment:
`networking.firewall.allowedTCPPorts = mkIf cfg.openFirewall [ cfg.port ];` looks better to me

Reply:
sure does the same thing. Please look at it again

Reply:
the CI error seems weird to me. Any idea? I applied your suggestions and rebased.

Reply:
this is my bad: see #203952 for the fix.

Reply:
so i guess this is fixed now. lets try again.